HW2/P5/P5.txt

```diff
@@ -0,0 +1 @@
+The biggest improvement in frames per second that I saw was a result of using the grid to get neighbors instead of comparing all pairs of balls. As the specs said, this resulted in an improvement from O(n^2) to O(n) which was a massive improvement, espeically when the number of balls gets increased. The only improvements, such as multithreading by changing range to prange did not provide nearly as impressive frames per second improvements as creating the grid. 
```

Comment:
Do you have the run time comparisons of the different methods? (threading and different spatial reasoning?)